MaintenanceManager/MaintenanceManager.cpp

```diff
@@ -222,11 +222,9 @@ namespace WPEFramework {
 
             LOGINFO("Reboot_Pending :%s",g_is_reboot_pending.c_str());
 
-#if defined (SKY_BUILD)
-            bool internetConnectStatus = true;
-#else
-            bool internetConnectStatus = isDeviceOnline();
-#endif
+            bool internetConnectStatus = false;
```

Comment:
This was commented on sky platforms to address https://ccp.sys.comcast.net/browse/XIONE-5622. Removing this will introduce XIONE-5622 issue again.

Reply:
as per the discussion with the ARCH team, this was another suggestion. Closing this PR.